ehitchhiking-ui/src/app/shared/services/data.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
```

Comment:
It is better to use separate service for each modal/page. Rename it

Reply:
renamed